Cour-de-cassation/label
# Feat/pre assignation

## Issue description :
You can set a pre-assignation for a document, after its annotation, the document is automatically assigned to the specified user

## Describe your changes :
- add a new admin page for CRUD preAssignations
- add backend part

## How to test :
- create a pre-assignation
- import a document and annotate
- check that the document is correctly assigned

## Checklist before requesting a review
> Les documents pourront être pré-assignés dans Label, une fois importés et annotés, il seront assignés directement a l'utilisateur désigné.
